app/assets/javascripts/lists.js

```diff
@@ -0,0 +1,11 @@
+$(function() {
+  $('.wishlist-remove').hide();
+
+  $('form').find(':checkbox').bind('change', function() {
+    if ($('input:checkbox:checked').length) {
+      $('.wishlist-remove').show();
+    } else {
+      $('.wishlist-remove').hide();
```

Comment:
We should mark it as disabled dynamically instead of hiding/showing the entire button.